pkg/controller/testutil/test_utils.go

```diff
@@ -21,6 +21,7 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
+	"golang.org/x/exp/slices"
```

Comment:
Please group the import packages.

Reply:
Moved to a different group. What is the convention for grouping packages? Not sure if there is something kubernetes specific